lib/raidenclient/RaidenClient.ts

```diff
@@ -166,15 +165,11 @@ class RaidenClient extends BaseClient {
    */
   private sendRequest = (endpoint: string, method: string, payload?: object): Promise<http.IncomingMessage> => {
     return new Promise((resolve, reject) => {
-      if (!this.isConnected()) {
```

Comment:
We may still want to have a check here to verify that we are connected, right? Even though it wouldn't necessarily be disabled.

Reply:
I thought about keeping it, but the `verifyConnection` also uses `sendRequest` internally - making it fail because the client is not connected. I keep pondering whether this is the correct place to do this check. Analogically, `lndClient` doesn't have such safeguards.

Reply:
OK that makes sense, I think I may have been confusing this method with `sendPayment` actually. This isn't the place for the check, thanks.